storage/cache/subtree_cache.go

```diff
@@ -246,6 +251,7 @@ func (s *SubtreeCache) getNodeHashUnderLock(id storage.NodeID, getSubtree GetSub
 	prefixKey := string(px)
 	c := s.subtrees[prefixKey]
 	if c == nil {
+		glog.Warningf("Cache miss for %x so we'll try to fetch from storage", prefixKey)
```

Comment:
Probably don't want a Warning here - that typically flushes to disk (or at least it does in the C++ glog)